encoder/vaapiencoder_hevc.cpp

```diff
@@ -1022,16 +1023,7 @@ Encode_Status VaapiEncoderHEVC::setParameters(VideoParamConfigType type, Yami_PT
     case VideoParamsTypeAVC: {
```

Comment:
Suppose you are a user. Who did not read the code. Just want know which configure can apply to HEVC. this name may make him confuse.